src/main/java/org/embulk/util/config/Version.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2020 The Embulk project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.embulk.util.config;
+
+import java.net.URL;
+import java.util.jar.Attributes;
+import java.util.jar.JarFile;
+import java.util.jar.Manifest;
+
+/**
+ * A utility class defining a constant of Jackson's Module version.
+ */
+final class Version {
+    static final com.fasterxml.jackson.core.Version JACKSON_MODULE_VERSION = getVersion();
+
+    private Version() {
+        // No instantiation.
+    }
+
+    static com.fasterxml.jackson.core.Version parseVersionForTesting(final String version) {
+        return parseVersion(version);
+    }
+
+    private static com.fasterxml.jackson.core.Version getVersion() {
+        try {
+            return parseVersion(getImplementationVersion());
+        } catch (final Throwable any) {  // It must not fail.
```

Comment:
Not sure about the intention of this comment, because it can fail if `getImplementationVersion` returns `null`.

Reply:
Looks okay to me. Thanks for explaining.